inspire/base/templates/search/collection_hep.html

```diff
@@ -0,0 +1,84 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{%- extends "search/collection.html" -%}
+
+{% block inner_content %}
+  {% block collection_header %}
+    <div id="collection-search-bar">
+      <div class="title" id="hep">
+        Search 1.117.977 articles
+      </div>
+      <div id="search-bar">
+        {{ search_bar("hep") | safe }}
+      </div>
+      <div class="subtitle">
+        Example queries:
+        <a href="/search?p=arXiv%3A1007.5048&cc=HEP">arXiv:1007.5048</a>
+        | 
+        <a href="/search?p=find+a+ellis+and+refersto+a+witten&cc=HEP">find a ellis and refersto a witten</a>
+      </div>
+    </div>
+  {% endblock %}
+
+  {% block collection_how_to_search %}
```

Comment:
I guess you don't need the blocks here if there is no child template that will overwrite them.